paddle/fluid/framework/ir/graph_pattern_detector.cc

```diff
@@ -2047,10 +2047,11 @@ PDNode *patterns::Reshape2Matmul::operator()() {
 }
 
 PDNode *patterns::MatmulWithInputOps::operator()(bool with_residual) {
```

Comment:
As inputs are removed from this pattern, maybe base `patterns::Matmul` could be adjusted and used?

Reply:
All in all, if I move the change to fused_matmul to the placement pass, onyl fused_matmul will be needed here, and it will be the same with other operators.